internal/clusterfeature/interface.go

```diff
@@ -93,12 +93,14 @@ type FeatureManager interface {
 
 	// Name returns the feature's name.
 	Name() string
+
+	BeforeSave(ctx context.Context, clusterID uint, spec FeatureSpec) (FeatureSpec, error)
```

Comment:
Why?

Reply:
After token refactor we don't need this function anymore, i delete it